C/cufinufft/build_tarballs.jl

```diff
@@ -0,0 +1,83 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using BinaryBuilderBase
+include(joinpath(@__DIR__, "..", "..", "fancy_toys.jl"))
+include(joinpath(@__DIR__, "..", "..", "platforms", "cuda.jl"))
+
+# Build script for the CUDA part of FINUFFT
+
+# Builds for all compatible CUDA platforms, but without microarchitecture expansion (not
+# needed for CUDA cuda, and would produce a giant amount of artifacts)
+name = "cufinufft"
+version = v"2.2.0"
+
+commit_hash = "51892059a4b457a99a2569ac11e9e91cd2e289e7";
+preferred_gcc_version=v"11"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/flatironinstitute/finufft.git", commit_hash)
+]
+
+# Build script: cufinufft, all possible archs available for each CUDA version
+# - CMake toolchain looks for compiler in CUDA_PATH/bin/nvcc
+#   and libs in CUDA_PATH/lib64, so create link.
+script = raw"""
+cd $WORKSPACE/srcdir/finufft*/
+
+export CUDA_PATH="$prefix/cuda"
+ln -s $prefix/cuda/lib $prefix/cuda/lib64
```

Comment:
I think you need to delete this symlink at the end, otherwise it ends up with a broken symlink in the tarball